src/components/shared/AddRemoveForm.js

```diff
@@ -86,7 +86,7 @@ class AddRemoveForm extends React.Component {
     return (
       <form ref="form" onSubmit={this.onSubmit}>
         <div className="field">
-          <h2 className="subtitle">{title} {this.getErrorMessage()} <Buttons.RecentlyUpdated onToggle={this.toggleRecentOnly} recentOnly={recentOnly} /></h2>
+          <h2 className="subtitle"><span data-test="title">{title}</span> {this.getErrorMessage()} <Buttons.RecentlyUpdated onToggle={this.toggleRecentOnly} recentOnly={recentOnly} /></h2>
```

Comment:
I like to keep my HTML elements to a minimum but this one is needed for cleaner testing. If we tried to grab the whole subtitle, we would get additional component code and possibly an error message. This will allow us to test each part of the subtitle separately.